ui/qt/src/Ariadne/UI/Qt/MainWindow.hs

```diff
@@ -42,29 +42,33 @@ initMainWindow :: UiLangFace -> UiHistoryFace -> IO MainWindow
 initMainWindow langFace historyFace = do
   mainWindow <- QMainWindow.new
   QWidget.setWindowTitle mainWindow ("Ariadne" :: String)
+  QWidget.resizeRaw mainWindow 960 640
 
   (qWallet, wallet) <- initWallet langFace
   (qRepl, repl) <- initRepl langFace historyFace
-  (qMenuBar, menuBar) <- initMenuBar
+  (qTopBar, topBar) <- initTopBar
   (qLogs, logs) <- initLogs
   (qHelp, help) <- initHelp langFace
 
-  QMainWindow.setMenuBar mainWindow qMenuBar
   QWidget.setParentWithFlags qLogs mainWindow Dialog
   QWidget.setParentWithFlags qHelp mainWindow Dialog
 
   mainLayout <- QVBoxLayout.new
-  QBoxLayout.addWidget mainLayout qWallet
-  QBoxLayout.addLayout mainLayout qRepl
-  QBoxLayout.setStretch mainLayout 0 2
-  QBoxLayout.setStretch mainLayout 1 1
+  QLayout.setContentsMarginsRaw mainLayout 0 0 0 0
+  QLayout.setSpacing mainLayout 0
+
+  QBoxLayout.addLayout mainLayout qTopBar
+  QBoxLayout.addLayout mainLayout qWallet
+  QBoxLayout.addWidget mainLayout qRepl
+
+  QBoxLayout.setStretch mainLayout 0 0
+  QBoxLayout.setStretch mainLayout 1 545
+  QBoxLayout.setStretch mainLayout 2 266
```

Comment:
Too magic, why not just `2` and `1`?

Reply:
I tried to follow the design as closely as possible :man_shrugging: